cloud-optimized-geotiffs/cogs-overview_resampling.ipynb

```diff
@@ -343,28 +343,43 @@
     "    return dst_path"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "8577e1d8",
+   "metadata": {},
+   "source": [
+    "Now lets make a list of the Resampling methods offered by GDAL. Some resampling methods aren't appropriate for the data, so we are doing to drop those from the list. You can see descriptions at https://gdal.org/programs/gdalwarp.html#cmdoption-gdalwarp-r "
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 1,
    "id": "67cb709e",
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "['nearest', 'bilinear', 'cubic', 'cubic_spline', 'lanczos', 'average', 'mode', 'gauss', 'rms']\n"
+      "['nearest', 'bilinear', 'cubic', 'cubic_spline', 'lanczos', 'average', 'mode', 'gauss', 'sum', 'rms']\n"
      ]
     }
    ],
    "source": [
     "# Make a list of resampling methods that GDAL 3.4+ allows\n",
     "\n",
-    "from rasterio.enums import Resampling as ResamplingEnums\n",
+    "from rasterio.enums import Resampling\n",
     "\n",
     "# Drop some irrelevant methods\n",
-    "OverviewResampling = [r for r in ResamplingEnums if (r.value < 8) or r.value > 13]\n",
-    "resample_methods = [r.name for r in OverviewResampling]\n",
+    "excluded_resamplings = {\n",
+    "    Resampling.max,\n",
+    "    Resampling.min,\n",
+    "    Resampling.med,\n",
+    "    Resampling.q1,\n",
+    "    Resampling.q3,\n",
+    "}\n",
+    "OverviewResampling = [method for method in Resampling if method not in excluded_resamplings]\n",
```

Comment:
small (optional) nit: use snake_case instead of camel case for `overview_resamplings`?

Reply:
now I'm going to check everything